views/probationDocuments.njk

```diff
@@ -43,38 +42,86 @@
   <p>The Delius record should have the prison number assigned to the Delius NOMS number and the Throughcare (custody) details may need to be updated.</p>
 {% endmacro %}
 
+{% macro documentDetail(document) %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <a class="qa-document-link" href="#">{{ document.documentName }}</a>
+    </div>
+    <div class="govuk-grid-column-full">
+      {{ document.type}} created by {{ document.author }}
+    </div>
+    {% if document.description %}
+      <div class="govuk-grid-column-full govuk-caption-m">
+        {{ document.description}}
+      </div>
+    {%  endif %}
+  </div>
+{% endmacro %}
+
+{% macro documentTable(documents, caption) %}
```

Comment:
this is a more practical way of building tables than what we've been using - we've been falling back to not using the macros as didn't want to do data transforming of of rows in the controller/custom filters (especially when we need to render buttons and things)